xchange-coinmate/src/main/java/org/knowm/xchange/coinmate/dto/trade/CoinmateOrder.java

```diff
@@ -0,0 +1,83 @@
+package org.knowm.xchange.coinmate.dto.trade;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.math.BigDecimal;
+
+public class CoinmateOrder {
+
+  private final long id;
+  private final long timestamp;
+  private final String type;
+  private final BigDecimal price;
+  private final BigDecimal remainingAmount;
+  private final BigDecimal originalAmount;
+  private final BigDecimal stopPrice;
+  private final String status;
+  private final String orderTradeType;
+  private final BigDecimal avgPrice;
+
+  public CoinmateOrder(
```

Comment:
should currency pair not be added into this as well?

Reply:
Unfortunately the API does not provide currency pair here.